test/CMakeLists.txt

```diff
@@ -552,30 +555,25 @@ if (HDF5_ENABLE_FORMATTERS)
   clang_format (HDF5_TEST_thread_id_FORMAT thread_id)
 endif ()
 
+if (HDF5_BUILD_UTILS)
 #-- Adding test for mirror_vfd
 add_executable (mirror_vfd ${mirror_vfd_SOURCES})
-target_include_directories (mirror_vfd PRIVATE "${HDF5_SRC_DIR};${HDF5_SRC_BINARY_DIR};$<$<BOOL:${HDF5_ENABLE_PARALLEL}>:${MPI_C_INCLUDE_DIRS}>")
-if (NOT BUILD_SHARED_LIBS)
-  TARGET_C_PROPERTIES (mirror_vfd STATIC)
-  target_link_libraries (mirror_vfd PRIVATE ${HDF5_TEST_LIB_TARGET})
-else ()
-  TARGET_C_PROPERTIES (mirror_vfd SHARED)
-  target_link_libraries (mirror_vfd PRIVATE ${HDF5_TEST_LIBSH_TARGET})
-endif ()
-set_target_properties (mirror_vfd PROPERTIES FOLDER test)
-
-#-----------------------------------------------------------------------------
-# Add Target to clang-format
-#-----------------------------------------------------------------------------
-if (HDF5_ENABLE_FORMATTERS)
-  clang_format (HDF5_TEST_mirror_vfd_FORMAT mirror_vfd)
-endif ()
+  target_include_directories (mirror_vfd PRIVATE "${HDF5_SRC_DIR};${HDF5_SRC_BINARY_DIR};$<$<BOOL:${HDF5_ENABLE_PARALLEL}>:${MPI_C_INCLUDE_DIRS}>")
+  if (NOT BUILD_SHARED_LIBS)
+    TARGET_C_PROPERTIES (mirror_vfd STATIC)
+    target_link_libraries (mirror_vfd PRIVATE ${HDF5_TEST_LIB_TARGET})
+  else ()
+    TARGET_C_PROPERTIES (mirror_vfd SHARED)
+    target_link_libraries (mirror_vfd PRIVATE ${HDF5_TEST_LIBSH_TARGET})
+  endif ()
+  set_target_properties (mirror_vfd PROPERTIES FOLDER test)
 
-#-----------------------------------------------------------------------------
-# Add Target to clang-format
-#-----------------------------------------------------------------------------
-if (HDF5_ENABLE_FORMATTERS)
-  clang_format (HDF5_TEST_event_set_FORMAT event_set)
```

Comment:
Looks like this line for the separate event_set test got accidentally removed

Reply:
Not accident it is included in the make macro as event_set is in the list